### ISSUE TITLE
Getting an Abort Trap 6 error on Mac

### ISSUE THREAD

**Issue by SSPharoah** [cwmonkey/nms-expeditions#14]
Hello!

I'm crashing to desktop whenever I try to take a photograph in photo mode for The Omega Expedition, Phase 3, Element 2, "Grand Tour - Photograph a range of planetary environments 0/3."

This occurs immediately after pressing the mouse button to take a photo, regardless of where I am.  From the ship, standing in a station, or planetside.  

I've attached my crashdump, if you're interested.  Either way, great work and thank you!  I'm having a ball playing through all the older expeditions I've missed.  Please forgive if I've broken any rules of GitHub etiquette.  I'm pretty new here.

Thanks again!
[CRASHDUMP.pdf](https://github.com/user-attachments/files/16368902/CRASHDUMP.pdf)

**Comment by cwmonkey** [comment]
@SSPharoah I just hopped into Omega and I was able to take a picture of a frozen planet and an airless planet in the first system with no issues (on the ground and in my ship). My best guess is you may have a mod installed that is causing an issue (I have no mods installed, and the game is updated to the latest Worlds patch.

Unfortunately I'm not adept at reading crash logs so I couldn't get anything from there. You may want to try taking pictures on a regular save after making sure the game is up-to-date. You can also try verifying the integrity of the files in Steam and/or reinstalling the game.

If you have any issues with no mods and a clean install files then I'll see if I can think of anything else to try.

**Comment by SSPharoah** [comment]
No mods, but it's also happening with my other save files, so I switched the old JSON back in, and it's still happening.  In light of this, I'm starting to think this would have happened before I even installed your JSON.  Who knows how long it's been since I took a picture in-game (before being specifically requested to do so).  I think I'm going to reinstall.  My guess is something completely unrelated got corrupted.  I appreciate your attention to this though.  The community is lucky to have you.
